$MirOS: ports/www/firesomething/patches/patch-browser_app_profile_firefox_js,v 1.2 2008/12/05 16:26:08 bsiegert Exp $
--- browser/app/profile/firefox.js.orig	Tue Nov 18 22:35:57 2008
+++ browser/app/profile/firefox.js	Wed Dec 17 23:21:26 2008
@@ -89,12 +89,12 @@ pref("browser.dictionaries.download.url"
 // App-specific update preferences
 
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);
+pref("app.update.enabled", false);
 
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make
 // the UI easier to construct.
-pref("app.update.auto", true);
+pref("app.update.auto", false);
 
 // Defines how the Application Update Service notifies the user about updates:
 //
@@ -188,7 +188,7 @@ pref("browser.shell.checkDefaultBrowser"
 
 // 0 = blank, 1 = home (browser.startup.homepage), 2 = last visited page, 3 = resume previous browser session
 // The behavior of option 3 is detailed at: http://wiki.mozilla.org/Session_Restore
-pref("browser.startup.page",                1);
+pref("browser.startup.page",                0);
 pref("browser.startup.homepage",            "resource:/browserconfig.properties");
 
 pref("browser.cache.disk.capacity",         50000);
@@ -590,7 +590,7 @@ pref("gecko.handlerService.allowRegister
 
 #ifdef MOZ_SAFE_BROWSING
 // Safe browsing does nothing unless this pref is set
-pref("browser.safebrowsing.enabled", true);
+pref("browser.safebrowsing.enabled", false);
 
 // Prevent loading of pages identified as malware
 pref("browser.safebrowsing.malware.enabled", true);
@@ -656,12 +656,9 @@ pref("browser.safebrowsing.malware.repor
 
 pref("browser.EULA.version", 3);
 pref("browser.rights.version", 3);
-pref("browser.rights.3.shown", false);
+pref("browser.rights.3.shown", true);
 
-#ifdef DEBUG
-// Don't show the about:rights notification in debug builds.
 pref("browser.rights.override", true);
-#endif
 
 pref("browser.sessionstore.enabled", true);
 pref("browser.sessionstore.resume_from_crash", true);
@@ -737,7 +734,7 @@ pref("places.frecency.unvisitedTypedBonu
 // 0 - don't pre-populate anything
 // 1 - pre-populate site URL, but don't fetch certificate
 // 2 - pre-populate site URL and pre-fetch certificate
-pref("browser.ssl_override_behavior", 1);
+pref("browser.ssl_override_behavior", 2);
 
 // Controls the display of domain in the identity box for SSL connections.
 // 0 - do not show domain
